scripts/detect_jumps.py

```diff
@@ -1,12 +1,18 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
 
 import sys
 from collections import defaultdict, deque
 import traceback
-from itertools import ifilterfalse
+if sys.version_info.major == 2:
+    from itertools import ifilterfalse
+else:
+    from itertools import filterfalse as ifiterfalse
 
 def hex_reverse(data):
-    return ''.join(reversed([data[i:i+2] for i in xrange(0, len(data), 2)]))
+    if sys.version_info.major == 2:
+        return ''.join(reversed([data[i:i+2] for i in xrange(0, len(data), 2)]))
+    else:
+        return ''.join(reversed([data[i:i+2] for i in range(0, len(data), 2)]))
```

Comment:
I guess you can solve this with a similar import trick - see above
or use something from here: https://python-future.org/compatible_idioms.html#lists-versus-iterators